src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
+    const progressNum = + ( ( mainStat / goal ) * 100 ).toFixed(2)
+    return ( progressNum ).toString().concat('%')
+  }
+
+  render() {
+
+    const { title, x, y, mainStat, goal } = this.props
+    const positionStyle = { top: x, left: y }
+    const progressString = this.calculateProgress( mainStat, goal )
+    const isComplete = mainStat / goal >= 1
+
+    return (
+      <article>
+        <div className="widget widget-width-1 widget-height-1" style={ positionStyle }>
+          <Header title={ title }/>
```

Comment:
Should changing the Header component and all of the other widgets be part of another issue or part of this PR? 
We don't want to submit code that would be broken if merged.